feat: add `math/base/special/acscd` (#1789)
stdlib-js/stdlib: lib/node_modules/@stdlib/math/base/special/acscd/docs/repl.txt

{{alias}}( x )

>     Computes the arccosecant of a double-precision floating-point number.

Reviewer: ```suggestion
    Computes the arccosecant (in degrees) of a double-precision floating-point
    number.
```

>     If input does not satisfies `x >= 1` or `x <= -1` the function returns NaN.

Reviewer: ```suggestion
    If `x` does not satisfy `x >= 1` or `x <= -1`, the function returns NaN.
```

    Parameters
    ----------
    x: number
        Input value.

    Returns
    -------
    y: number

>         Arccosecant (in degree).

Reviewer: ```suggestion
        Arccosecant (in degrees).
```
